osu.Game/Skinning/Editor/SkinEditor.cs

```diff
@@ -165,10 +165,22 @@ private void skinChanged()
 
         private void placeComponent(Type type)
         {
+            var targetContainer = getTarget(SkinnableTarget.MainHUDComponents);
+
+            if (targetContainer == null)
+                return;
+
             if (!(Activator.CreateInstance(type) is ISkinnableDrawable component))
                 throw new InvalidOperationException($"Attempted to instantiate a component for placement which was not an {typeof(ISkinnableDrawable)}.");
 
-            getTarget(SkinnableTarget.MainHUDComponents)?.Add(component);
+            var drawableComponent = (Drawable)component;
+
+            // give newly added components a sane starting location.
+            drawableComponent.Origin = Anchor.TopCentre;
+            drawableComponent.Anchor = Anchor.TopCentre;
+            drawableComponent.Y = targetContainer.DrawSize.Y / 2;
```

Comment:
Is there a reason you did it this way, as opposed to using `Centre`? I can only assume it's to avoid negative coordinates if the user is to then move the component to the top quadrants of the screen, but if you wanted to void that then you'd use `TopLeft` anyway.

Reply:
I mentioned this in the PR opening description.

Reply:
Hm, yeah I dunno. Let's go with it for now.

Reply:
To elaborate: there's basically zero use case where centre is correct. So i'd rather have the user start with topcentre and then work from there to learn/correct things as they learn the systems.